### ISSUE TITLE
Accessible oc-switch

### ISSUE THREAD

**Issue by PVince81** [owncloud/owncloud-design-system#545]
The oc-switch is a div and one cannot tab into it to switch its state like a checkbox.

@marcus-herrmann FYI a topic for later on

unless we agree to get rid of it completely, see #482

**Comment by marcus-herrmann** [comment]
If it will be fixed it should be fixed this way: https://github.com/owncloud/owncloud-design-system/pull/525/files#diff-f940e824a69fbb52c4246d7ff35929fcR2

Another example of the same approach: https://github.com/marcus-herrmann/vue-switch-button

**Comment by pascalwengerter** [comment]
Should be resolved since #1421
